tests/unit/components/search/SearchForm.spec.js

```diff
@@ -266,4 +268,56 @@ describe('components/search/SearchForm', () => {
       // });
     });
   });
+
+  describe('mobileSearchOptions', () => {
```

Comment:
This suggests a function named thus. Change the text to 'mobile search buttons'.